convert2rhel/pkgmanager/handlers/yum/callback.py

```diff
@@ -66,7 +68,7 @@
 
 # We need to double inherit here, both from the callback class and the base
 # object class, just to initialize properly with `super`
-class PackageDownloadCallback(pkgmanager.DownloadProgress, object):
+class PackageDownloadCallback(pkgmanager.DownloadProgress, object):  # pylint: disable=useless-object-inheritance
```

Comment:
We need the object inheritance here because of the first inheritance, that is from yum lib?

Reply:
Correct.  The yum object is an old-style class on Python2 so we need to also explicitly inherit from object so our class is a new-style class and we can use `super`.